src/main/java/com/yahoo/sketches/concurrent/theta/ConcurrentUpdateSketch.java

```diff
@@ -0,0 +1,85 @@
+package com.yahoo.sketches.concurrent.theta;
+
+import com.yahoo.sketches.theta.Sketch;
+import com.yahoo.sketches.theta.UpdateSketch;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * @author eshcar
+ */
+public class ConcurrentUpdateSketch extends UpdateSketchComposition {
+
+  private static int PARALLELISM_LEVEL = 3;
+  protected static ExecutorService PROPAGATION_EXECUTOR_SERVICE = Executors.newWorkStealingPool
```

Comment:
Does the DataSketches package have its own executor pools?

Reply:
Maybe there is, I'm not sure.
I couldn't find ExecutorService in the package.

Reply:
The DS package does not have executor pools because the entire package was designed to be single threaded... until now :)